### ISSUE TITLE
Error requesting jobs

### ISSUE THREAD

**Issue by tmyroadctfig** [NSWSESMembers/libbeacon#1]
I'm sure this is something to do with my code or setup, but I'm getting an error trying to fetch jobs from beacon. Strangely, if I run the test suite everything seems to work fine.

Here's the error I'm seeing:
```$ node beacon-data.js 
Error: null
Success: true
_http_outgoing.js:355
    throw new Error('"value" required in setHeader("' + name + '", value)');
    ^

Error: "value" required in setHeader("User-Agent", value)
    at ClientRequest.OutgoingMessage.setHeader (_http_outgoing.js:355:11)
    at new ClientRequest (_http_client.js:85:14)
    at Object.exports.request (http.js:31:10)
    at Object.exports.request (https.js:199:15)
    at Request.start (/home/luke/work/beacon-data/node_modules/request/request.js:708:30)
    at Request.end (/home/luke/work/beacon-data/node_modules/request/request.js:1405:28)
    at end (/home/luke/work/beacon-data/node_modules/request/request.js:433:16)
    at /home/luke/work/beacon-data/node_modules/request/request.js:447:9
    at _combinedTickCallback (internal/process/next_tick.js:73:7)
    at process._tickCallback (internal/process/next_tick.js:104:9)
```

Here's the code:
```var Libbeacon = require('libbeacon');

var beacon = new Libbeacon();
beacon.login('beacon-id', 'password', function(err, success) {
    console.log("Error: " + err);
    console.log("Success: " + success);

    beacon.get('Jobs/1', {}, function(error, data) {
        console.log(data);
      });
});
```

If you can spot anything that I'm doing wrong I'd be appreciative. Thanks, Luke

**Comment by sdunster** [comment]
uhh pretty sure @OSPFNeighbour has a fix

**Comment by OSPFNeighbour** [comment]
Add this after line 156 in libbeacon/lib/main.js

options.headers['User-Agent'] = this._userAgent;

**Comment by tmyroadctfig** [comment]
Thanks Tim, that fixed it for me. That line seemed to be in my copy from npm already, but using userAgent instead of _userAgent? Anyway I've added it in with a pull request.

**Comment by sdunster** [comment]
Fixed in #2
